student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person_spec'
+
+class Student < Person
+  def initialize(name, age, parent_permission, classroom)
+    super(name, age, parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    puts('¯(ツ)/¯')
+  end
```

Comment:
- [ ] As part of the project requirements the string should be returned upon invocation of the `play_hooky` method not printed.